### ISSUE TITLE
No check on existence of packages in modules

### ISSUE THREAD

**Issue by emilyzheng** [release-engineering/ubi-population-tool#27]
Packages which are in package whitelist and not existing in source repo won't be copied to ubi repo, but packages included in modules will be, even if they don't exist in source repo. E.g. there's ubipop log:
```
2019-03-27 05:25:32 +0000 [INFO    ] Associating httpd-tools-2.4.37-4.module+el8+2248+23d5e2f2.x86_64.rpm from rhel-7-server-rpms__7Server__x86_64 to ubi-7-server-rpms__7Server__x86_64
```
httpd-tools is a package in httpd module. It won't really be associated to ubi-7-server-rpms__7Server__x86_64 when it doesn't exist in rhel-7-server-rpms__7Server__x86_64.

Above log is misleading since it says the rpm will go to ubi repo, but actually it won't, although this may not be harmful to final content in ubi repo.

**Comment by emilyzheng** [comment]
Similar issue on source rpm/debuginfo rpm, they may be outputted as associated to ubi repos, but not even exist in source rpm repo/debug rpm repo.

**Comment by alexandrevicenzi** [comment]
@emilyzheng @rbikar it would be nice to show a warning if a package is missing, otherwise, the module is kinda useless. Let's suppose httpd module, it makes no sense to install it if mod_http2 package is missing.

Or even, the association of a module should fail if packages are missing.

**Comment by rohanpm** [comment]
It sounds like the assumption there is that anything in modulemd "artifacts" must be available on one of the system's repos for the module to be usable, but that's not the case.  A simple example is that modulemd artifacts have references to debuginfo/debugsource RPMs, but users don't have debuginfo repos enabled by default (and most users will never enable them). Yet this doesn't interfere with installing or using the module.

Maybe there are also other non-debuginfo or non-source-RPM cases where it's expected for something to be listed in modulemd artifacts but not actually be present in repos enabled on the end-user system.  I don't know...

Unless you can find clear information about it somewhere I'd recommend only going as far as warnings, and not fatal errors, when a modulemd refers to an RPM which can't be found.

**Comment by alexandrevicenzi** [comment]
@rohanpm I agree with you, it's difficult to ensure, so better go only with a warning.

**Comment by alexandrevicenzi** [comment]
This task is related to #18.

**Comment by rbikar** [comment]
Fixed by #91
